<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processing Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Controls">
      <item>
       <widget class="QPushButton" name="selectDirButton">
        <property name="text">
         <string>Select Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="processingMethodComboBox">
        <property name="toolTip">
         <string>Select processing method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="prevButton">
        <property name="text">
         <string>Previous</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Images">
      <item>
       <widget class="QGraphicsView" name="originalImageView">
        <property name="toolTip">
         <string>Original Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="upscaledImageView">
        <property name="toolTip">
         <string>Processed Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>